libs/optimizer/src/lib/optimizer.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { OptimizationResult, getOptimizer } from './optimizer';
+import {
+  createClosedFaiTriangleFixture,
+  createClosedFaiTriangleFixtureWithSmallCycle,
+  createClosedFaiTriangleFixtureWithSmallLoop,
+  createClosedFlatTriangleFixture,
+  createEmptyTrackFixture,
+  createFreeDistance1PointFixture,
+  createFreeDistance2PointsFixture,
+  createFreeDistance3PointsFixture,
+  createFreeDistanceFixture,
+  OptimizerFixture,
+} from './fixtures/optimizer.fixtures';
+import { ScoringRules } from './scoringRules';
+
+describe('optimizer', () => {
+  describe('given an empty request', () => {
+    const fixture = createEmptyTrackFixture();
+    it('should return a 0 score', () => {
+      expectOptimizationIsAsExpected(fixture);
+    });
+  });
+  [
+    ScoringRules.CzechEuropean,
+    ScoringRules.CzechLocal,
+    ScoringRules.CzechOutsideEurope,
+    ScoringRules.FederationFrancaiseVolLibre,
+    ScoringRules.Leonardo,
+    ScoringRules.Norway,
+    ScoringRules.UnitedKingdomClub,
+    ScoringRules.UnitedKingdomInternational,
+    ScoringRules.UnitedKingdomNational,
+    ScoringRules.XContest,
+    ScoringRules.XContestPPG,
+    ScoringRules.WorldXC,
+  ].forEach((rules) => {
+    describe(`${ScoringRules[rules]} rules`, () => {
+      const oneSegmentPerBranch = 1;
+      const tenSegmentsPerBranch = 10;
+      [oneSegmentPerBranch, tenSegmentsPerBranch].forEach((nbSegmentsPerBranch) => {
+        describe(`given a free distance request (${nbSegmentsPerBranch} segments(s)/branch)`, () => {
```

Comment:
remove `(s)` - `segments` is good enough

Reply:
Ok. Changed